zonemaster/zonemaster-backend
# Reinstate old RPC API methods

## Purpose

Remove deprecation warning and comments to all old RPC API methods and mark the new methods as experimental.
What were the deprecated methods (and old stable) are again the default stable API methods.

## Context

https://github.com/zonemaster/zonemaster/issues/1159

Relates to https://github.com/zonemaster/zonemaster-backend/pull/1054 and https://github.com/zonemaster/zonemaster-backend/pull/1083.

## Changes

Revert commits where old methods were deprecated and based upon new methods. Now the experimental methods are based on the stable one.

## How to test this PR

* Unit tests should pass
* Review the code and verify that the unit tests cover the stable API
* Check that the new experimental API is working (there is currently no unit test)
* Check that changes made in Backend for this release are still working (and have not been inadvertently reverted)
